pkg/build/graph/analysis/bc.go

```diff
@@ -4,30 +4,38 @@ import (
 	"fmt"
 	"sort"
 	"strings"
+	"time"
 
 	"github.com/gonum/graph"
 	"github.com/gonum/graph/topo"
 
+	"k8s.io/kubernetes/pkg/api/unversioned"
+
 	osgraph "github.com/openshift/origin/pkg/api/graph"
 	buildapi "github.com/openshift/origin/pkg/build/api"
 	buildedges "github.com/openshift/origin/pkg/build/graph"
 	buildgraph "github.com/openshift/origin/pkg/build/graph/nodes"
+	imageapi "github.com/openshift/origin/pkg/image/api"
 	imageedges "github.com/openshift/origin/pkg/image/graph"
 	imagegraph "github.com/openshift/origin/pkg/image/graph/nodes"
 )
 
 const (
-	TagNotAvailableWarning     = "ImageStreamTagNotAvailable"
-	LatestBuildFailedErr       = "LatestBuildFailed"
-	MissingRequiredRegistryErr = "MissingRequiredRegistry"
-	MissingImageStreamErr      = "MissingImageStream"
-	CyclicBuildConfigWarning   = "CyclicBuildConfig"
+	TagNotAvailableWarning         = "ImageStreamTagNotAvailable"
+	LatestBuildFailedErr           = "LatestBuildFailed"
+	MissingRequiredRegistryErr     = "MissingRequiredRegistry"
+	MissingOutputImageStreamErr    = "MissingOutputImageStream"
+	CyclicBuildConfigWarning       = "CyclicBuildConfig"
+	MissingImageStreamTagWarning   = "MissingImageStreamTag"
```

Comment:
(but ignore me if I'm lacking context here... long time I don't look at what this PR is doing)